src/backend/marsha/core/migrations/0032_auto_20220104_1135.py

```diff
@@ -0,0 +1,53 @@
+# Generated by Django 3.2.10 on 2022-01-04 11:35
+
+import django.contrib.postgres.fields
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("core", "0031_auto_20211220_1203"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="video",
+            name="allow_recording",
+            field=models.BooleanField(
+                default=True,
+                help_text="Allow video recording?",
+                verbose_name="allow video recording",
+            ),
+        ),
+        migrations.AddField(
+            model_name="video",
+            name="estimated_duration",
```

Comment:
Why this field is named `estimated_duration`? I mean why estimated ?

Reply:
This has been defined in #1243 
> When a live video is scheduled, we want to inform users of the estimated duration of the webinar

Reply:
As opposed to the actual duration of the live which something else. This value is just what the teacher wants to display on the live page during the registration period...